spec/models/arbitration_rule_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe ArbitrationRule do
+  describe '#action' do
+    it 'validates allowed actions' do
+      expect { FactoryGirl.create(:arbitration_rule, :action => 'skip') }
+        .to raise_error(ActiveRecord::RecordInvalid, /Action is not included in the list/)
+    end
+  end
+
+  describe '#xpression' do
+    it 'requires an expression to be set' do
+      expect { FactoryGirl.create(:arbitration_rule) }
```

Comment:
Minor, but I think it's generally better to have a factory that will create a valid model object by default, and to specify that the relevant attribute be blank in the body of this test here. It might be surprising for another developer to update the factory and find that this test fails.